packages/react-composites/src/composites/CallComposite/types/CallControlOptions.ts

```diff
@@ -66,30 +66,19 @@ export type CallControlOptions = {
 };
 
 /**
- * Placement for a custom button injected in the {@link CallControls}.
+ * Placement for a custom button injected in the {@link CallControls} and {@link CallWithChatControlBar}.
  *
- * 'first': Place the button on the left end (right end in rtl mode).
- * 'afterCameraButton': Place the button on the right (left in rtl mode) of the camera button.
- * ... and so on.
+ * 'mainBar': Place the button(s) on the right end of the center control bar but before the EndCallButton (left end in rtl mode).
+ * 'sideBar': Place the button(s) on the left end of the side control bar (right in rtl mode).
+ * 'overflowMenu': Place the buttons(s) on the end of the overflow Menu {@link MoreDrawer}
```

Comment:
Note `MoreDrawer` isn't public here so this `@link` won't work for external devs